modules/drivers/uart/src/rtos_uart_rx.c

```diff
@@ -44,6 +44,8 @@ static void uart_rx_error_callback(uart_callback_code_t callback_code, void * ap
 
 static void uart_rx_hil_thread(rtos_uart_rx_t *ctx)
 {
+    rtos_printf("UART Rx on tile %d core %d\n", THIS_XCORE_TILE, rtos_core_id_get());
```

Comment:
This is good but you should remove line 5 so printing is off by default
#define DEBUG_PRINT_ENABLE_RTOS_UART_RX 1

Reply:
Removed. There was a similar line in rtos_uart_tx.c, which has been removed too.